src/modules/control/src/ardu_adapter.py

```diff
@@ -65,30 +59,50 @@ def timer_callback(event):
         steering_angle = steering_msg 
 
     global desired_steering
-    desired_steering = ((-1)*steering_angle*180/pi+steering_analog_intercept)/steering_analog_slope+potentiometer_offset
-    
+    desired_steering = ((-1)*steering_angle*steering_analog_slope+steering_analog_intercept)
     steering_low_lvl_node_publisher.publish(desired_steering)
     driving_low_lvl_node_publisher.publish(desired_throttle)
+    
+    
+# def timer_callback(event):
```

Comment:
Will this be used for future implementations?

Reply:
Possibly, we still don't know the benefits of using timer callback compared to the other method of facilitating the ROS functionality.

Reply:
http://wiki.ros.org/ROS/Tutorials/WritingPublisherSubscriber(python)
This is the method for publisher/subscriber that I was going by.

Reply:
I think it is safe to stick with the method you have right now, and clean out the old code